tests/integration/test.basics.js

```diff
@@ -73,6 +73,20 @@ adapters.forEach(function (adapter) {
       return new PouchDB(dbs.name).destroy({});
     });
 
+    it('4339 throw useful error if method called on stale instance', function () {
+      var db = new PouchDB(dbs.name);
+
+      return db.put({
+        _id: 'cleanTest'
+      }).then(function () {
+        return db.destroy();
+      }).then(function () {
+        return db.get('cleanTest');
+      }).catch(function (err) {
+        should.equal(err instanceof Error, true, 'should be an error');
+      });
```

Comment:
I'm not following why you'd throw an error in the test, surely if the get succeeds then the tests should fail since it should always error in this circumstance?